test/integration/dbapi/conftest.py

```diff
@@ -152,13 +152,13 @@ def exasol_db(exasol_test_config, test_schema):
         password=config.password,
     )
 
-    connection.execute(f"DROP SCHEMA IF EXISTS {test_schema} CASCADE;")
-    connection.execute(f"CREATE SCHEMA {test_schema};")
+    connection.execute(f'DROP SCHEMA IF EXISTS "{test_schema}" CASCADE;')
+    connection.execute(f'CREATE SCHEMA "{test_schema}";')
```

Comment:
I think these need to be qingle quotes inside, not double quotes.

Reply:
You are right, and I think I got tricked by the auto formatter here ... xD